components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -78,6 +78,12 @@ - (void)presentationTransitionWillBegin {
   }
   bottomDrawerContainerViewController.shouldIncludeSafeAreaInContentHeight =
       self.shouldIncludeSafeAreaInContentHeight;
+  if (self.elevation > 0) {
+    bottomDrawerContainerViewController.elevation = self.elevation;
+  }
+  if (self.drawerShadowColor != nil) {
```

Comment:
Based on the setter below it looks like `nil` should be passed-through in this case as well.

Reply:
Removed checks.